source/client/options_impl.h

```diff
@@ -75,7 +75,9 @@ class OptionsImpl : public Options {
   uint32_t request_body_size_{0};
   envoy::api::v2::auth::UpstreamTlsContext tls_context_;
   uint32_t max_pending_requests_{1};
-  uint32_t max_active_requests_{largest_acceptable_uint32_option_value};
+  // This default is based the minimum recommendation for SETTINGS_MAX_CONCURRENT_STREAMS over at
+  // https://tools.ietf.org/html/rfc7540#section-6.5.2
+  uint32_t max_active_requests_{100};
```

Comment:
100?

Reply:
This header provides the canonical source for default values.
(We also need them in the grpc service, and we don't use TCLAP there)